osu.Game/Overlays/Mods/ModSearchContainer.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Overlays.Mods
+{
+    public partial class ModSearchContainer : SearchContainer
+    {
+        /// <summary>
+        /// A string that should match the <see cref="IFilterable"/> children
+        /// </summary>
+        public string ForcedSearchTerm
```

Comment:
Why is this required? It needs to be explained in the xmldoc as a remark, at very least, as it looks like a hack.

Reply:
Updated xmldoc